ihmc-graphics/src/libgdx/java/us/ihmc/rdx/input/ImGui3DViewInput.java

```diff
@@ -131,7 +131,8 @@ public Point3DReadOnly getPickPointInWorld(double fallbackXYPlaneIntersectionHei
             int columnAdjustment = aliasedMouseX * Float.BYTES;
             float normalizedDeviceCoordinateZ = 2.0f * depthBuffer.getFloat(rowAdjustment + columnAdjustment) - 1.0f;
 
-            if (normalizedDeviceCoordinateZ < 1.0f) // 1.0 corresponds to no collision
```

Comment:
On the first tick, `normalizedDeviceCoordinateZ` was -1.0, which is also not a collision, which initialized the footstep height to a point behind the camera, or something like that. The valid range is 0.0 -> 1.0.